CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## UNRELEASED
+
+* Check that the current user `canEdit` images before showing the `Generate an image with AI` button.
```

Comment:
This would be `canCreate` since (if I recall properly) this feature can only create new images.

Reply:
And so there is no reason to keep people who only have create permission out.